tests/unit/test_upstream.py

```diff
@@ -64,3 +65,37 @@ def test_create_pull(upstream_mock, upstream_pr_mock, fork_username):
         target_branch="test_target",
         fork_username=fork_username,
     )
+
+
+@pytest.mark.parametrize(
+    "action_config,result",
+    [
+        pytest.param("one cmd", [["one", "cmd"]], id="str_command"),
+        pytest.param(["one cmd"], [["one", "cmd"]], id="list_command"),
+        pytest.param([["one", "cmd"]], [["one", "cmd"]], id="list_in_list_command"),
+        pytest.param(
+            ["one cmd", "second cmd"],
+            [["one", "cmd"], ["second", "cmd"]],
+            id="two_str_commands_in_list",
+        ),
+        pytest.param(
+            [["one", "cmd"], ["second", "cmd"]],
+            [["one", "cmd"], ["second", "cmd"]],
+            id="two_list_commands_in_list",
+        ),
+        pytest.param(
+            [["one", "cmd"], "second cmd"],
+            [["one", "cmd"], ["second", "cmd"]],
```

Comment:
oh dear...

Reply:
I don't like this syntax as well, but some people are used to this from other tools...